src/main/scala/cc/factorie/app/nlp/ner/StackedChainNer.scala

```diff
@@ -585,55 +585,10 @@ class StackedChainNer[L<:NerTag](labelDomain: CategoricalDomain[String],
   }
 }
 
-object StackedChainNer {
-  def initLexicons() : Unit = synchronized {
-    lexicon.iesl.Month.toString()
-    lexicon.iesl.Day.toString()
-
-    lexicon.iesl.PersonFirst.toString()
-    lexicon.iesl.PersonFirstHigh.toString()
-    lexicon.iesl.PersonFirstHighest.toString()
-    lexicon.iesl.PersonFirstMedium.toString()
-
-    lexicon.iesl.PersonLast.toString()
-    lexicon.iesl.PersonLastHigh.toString()
-    lexicon.iesl.PersonLastHighest.toString()
-    lexicon.iesl.PersonLastMedium.toString()
-
-    lexicon.iesl.PersonHonorific.toString()
-
-    lexicon.iesl.Company.toString()
-    lexicon.iesl.JobTitle.toString()
-    lexicon.iesl.OrgSuffix.toString()
-
-    lexicon.iesl.Country.toString()
-    lexicon.iesl.City.toString()
-    lexicon.iesl.PlaceSuffix.toString()
-    lexicon.iesl.USState.toString()
-    lexicon.iesl.Continents.toString()
-
-    lexicon.wikipedia.Person.toString()
-    lexicon.wikipedia.Event.toString()
-    lexicon.wikipedia.Location.toString()
-    lexicon.wikipedia.Organization.toString()
-    lexicon.wikipedia.ManMadeThing.toString()
-    lexicon.iesl.Demonym.toString()
-
-    lexicon.wikipedia.Book.toString()
-    lexicon.wikipedia.Business.toString()
-    lexicon.wikipedia.Film.toString()
-
-    lexicon.wikipedia.LocationAndRedirect.toString()
-    lexicon.wikipedia.PersonAndRedirect.toString()
-    lexicon.wikipedia.OrganizationAndRedirect.toString()
-  }
-}
-
```

Comment:
Eager initialization is useful for running a service. For example if we have NER running in a web service then the lexicon initialisation can be run as part of the startup of the service, rather than the first input taking many multiples of the steady state behaviour. This particular use case is why the initLexicons method exists, as it was requested by people in my group.

Also in multithreaded environments the lazy loading construct in Scala isn't very good (i.e. it can deadlock the application, see http://docs.scala-lang.org/sips/pending/improved-lazy-val-initialization.html). I'm pretty sure the lexicon initialisation code itself is thread safe, I've done some work to improve it, and it's basically synchronised everywhere but multithreaded locking is hard.